tree/treeplayer/CMakeLists.txt

```diff
@@ -3,6 +3,10 @@
 # @author Pere Mato, CERN
 ############################################################################
 
+if (ARROW_FOUND)
+  include_directories(${ARROW_INCLUDE_DIR})
+endif()
```

Comment:
I just did what is done elsewhere in the codebase:

```bash
→ git grep target_include_directories | wc
      14      57    1563

→ git grep include_directories | wc
     190     440   17411
```

I think this should be taken care of centrally or proper instructions on how to do that should be provided. What is the target here? `TreePlayer`?

Reply:
Yes, there is a problem with rootcling, that's why in many places it needs to be `include_directories()`. However, we want to change that in the future, so that rootcling gets automatically the include directories from each target, and that's only possible with `target_include_directories()`. However, you can leave it as is and I will change it after it's merged.